[faq] Added 'advanced osm editing' section

map-per opened the pull request:
Added an 'advanced osm editing' section to the faq, so this page can be linked to in https://github.com/organicmaps/organicmaps/pull/7748.

See also  #203

biodranik commented on content/faq/editing/advanced-editing/index.md:
```diff
@@ -0,0 +1,71 @@
+
+### ID Editor
+
+ID is an easy-to-use, beginner-friendly editor for OpenStreetMap data. It runs in your browser, so no installation is required. You, however, need a computer, as the program is not optimized for smartphone screens. If you are new to OSM this is the editor to use.
```
Let's highlight that ID is for desktops only, and recommend other options for mobile users (they should go before details about how to use ID on the desktop.

biodranik commented on content/faq/editing/advanced-editing/index.md:
advanced-map-editing

biodranik commented on content/faq/editing/advanced-editing/index.md:
```diff
@@ -0,0 +1,71 @@
+---
+title: How can I do more advanced editing in OpenStreetMap?
+description: "Tutorial for editing OpenStreetMap with more advanced tools like ID"
```
It's better to focus this page on two simple things:
1. Teach new users about OSM.
2. Recommend other editors (focus on mobile ones!). And make a link to another FAQ page "How to fix map errors using web ID editor?", it may have different recommendations, including how to copy a link to the place from OM by clicking on coordinates (there is an osm link there too), and other details.

map-per replied:
The problem is, that there are no mobile editors that are good for beginners and can do "advanced" edits (see https://github.com/organicmaps/organicmaps/pull/7748#issuecomment-2035347759). So ID on a desktop PC is the only good option.

biodranik replied:
What "advanced" cases for new users, in your opinion, can't be handled with existing mobile editors?

Forcing all mobile users to "find and use desktop to edit the map" doesn't look like a good idea. Most new users won't need really "advanced" stuff, from what we see, users need something simpler, that just can't be added/edited yet in OM.
At least a list of user options is needed for mobiles. I would say, that Vespucci and Go! Map can cover most of the cases.

map-per replied:
I can't say anything about Go! Map, as it's ios only, but Vespucci is rather complicated to use. But mentioning it any way is probably a good idea, as it is the only android editor that can do "advanced" stuff like adding a new road.

Osm Go! and Every Door can only edit POIs and with StreetComplete you can only answer simple quests. So with "advanced" I meant doing more than just editing POIs. So e.g. adding a new road or a building shape.

Have you tried out mobile editors so far? And if yes, how was your experience?

biodranik replied:
"Go! Map" is good enough for mobile. Using it to draw ways and fix areas rarely.

Then the text in the app should be clearer: "Simple map editing apps like Every Door or StreetComplete, or more advanced Vespucci" for Android and "Every Door app for simple map edits, and more advanced Go! Map"

biodranik commented on content/faq/editing/advanced-editing/index.md:
```diff
@@ -0,0 +1,71 @@
+---
+
+[OpenStreetMap.org](https://www.openstreetmap.org) (OSM) is a database with map data for the entire world. Similar to Wikipedia the data can be used free of charge and is crowd-sourced by millions of volunteers.
+**Organic Maps** uses the OpenStreetMap.org data to generate beautiful and detailed maps for you. 
```
Maybe add a paragraph how/when map data is updated here or below.

map-per replied:
No, I think that rather belongs into a different faq

biodranik commented on content/faq/editing/advanced-editing/index.md:
```diff
@@ -0,0 +1,71 @@
+For editing OpenStreetMap with ID follow these steps:
+
+1. Create or log into your account at [OpenStreetMap.org](https://www.openstreetmap.org)
+2.  Browse to the location you want to edit on OpenStreetMap.org and click *Edit*
```
```suggestion
2. Browse to the location you want to edit on OpenStreetMap.org and click *Edit*
```

map-per replied:
> A direct "edit" link to osm can be inserted, right?

Using the OSMCarto map for zooming to the right location first is more convenient. On older devices zooming or moving the map in ID can be very slow on lower zoom levels.

> A hint on how to extract coordinates or osm URL from OM can be helpful

But then you have the coordinates in the clipboard of your phone and not on the desktop PC

biodranik replied:
We are trying to optimize here the UX for _completely new_ users, not aware of OSM, and who came from the mobile OM app. They need something really _simple_ to get involved. Later, they can learn more about OSM and about any advanced use cases.

biodranik commented on content/faq/editing/advanced-editing/index.md:
```diff
@@ -0,0 +1,71 @@
+
+For editing OpenStreetMap with ID follow these steps:
+
+1. Create or log into your account at [OpenStreetMap.org](https://www.openstreetmap.org)
```
```suggestion
1. Create a new account or log in at [OpenStreetMap.org](https://www.openstreetmap.org)
```

map-per commented on content/faq/editing/advanced-editing/index.md:
```diff
@@ -0,0 +1,71 @@
+---
+title: How can I do more advanced editing in OpenStreetMap?
```
Ok, but I would rather use "How can I do more advanced map editing?".

map-per commented:
I updated the faq page:
- I added mobile editor, but placed ID first as editing is easier on a bigger screen and with mouse and keyboard. 
- I left everyting in one faq page as scattering the text would make it hard to read

biodranik commented on content/faq/editing/advanced-map-editing/index.md:
```diff
@@ -0,0 +1,79 @@
+
+## OpenStreetMap Editors
+
+For editing OSM, there are several options. If you have a desktop PC at hand, it is recommended that you use the ID Editor. The editor is easy to use for beginners, and a bigger screen, mouse, and keyboard make editing easier. The editor runs in your browser, so no installation is required.
```
```suggestion
For editing OSM, there are several options. If you have a laptop or desktop computer at hand, it's better to use the [ID Editor](https://www.openstreetmap.org/edit) that runs in your browser. The ID Editor is easy for beginners, and a bigger screen, mouse, and keyboard make map editing easier.
```

biodranik commented on content/faq/editing/advanced-map-editing/index.md:
```diff
@@ -0,0 +1,79 @@
+
+For editing OSM, there are several options. If you have a desktop PC at hand, it is recommended that you use the ID Editor. The editor is easy to use for beginners, and a bigger screen, mouse, and keyboard make editing easier. The editor runs in your browser, so no installation is required.
+
+If you don't have a desktop PC at hand, you can use the mobile editors Go Map for ios or Vespucci for Android. Go Map is easy to use for beginners, while Vespucci targets more advanced users.
```
```suggestion
For advanced map editing from a mobile device, use [Go!! Map](https://apps.apple.com/us/app/go-map/id592990211) for iOS or [Vespucci](https://play.google.com/store/apps/details?id=de.blau.android) for Android. Go Map is easy for beginners, while Vespucci targets more advanced users.

For simpler edits with more fun, you may also try [Every Door app](https://every-door.app/) for iOS and Android and [StreetComplete app](https://streetcomplete.app/) for Android.
```

biodranik commented on content/faq/editing/advanced-map-editing/index.md:
```diff
@@ -0,0 +1,79 @@
+For editing OSM, there are several options. If you have a desktop PC at hand, it is recommended that you use the ID Editor. The editor is easy to use for beginners, and a bigger screen, mouse, and keyboard make editing easier. The editor runs in your browser, so no installation is required.
+
+If you don't have a desktop PC at hand, you can use the mobile editors Go Map for ios or Vespucci for Android. Go Map is easy to use for beginners, while Vespucci targets more advanced users.
+
+Please pick one of the three editors to proceede.
```
```suggestion
```

Following succinct OM style in FAQ is beneficial too )

biodranik commented on content/faq/editing/advanced-map-editing/index.md:
```diff
@@ -0,0 +1,79 @@
+For editing OpenStreetMap with ID follow these steps:
+
+1. Create a new account or log in at [OpenStreetMap.org](https://www.openstreetmap.org)
+2. Browse to the location you want to edit on OpenStreetMap.org and click *Edit*
```
```suggestion
2. Browse to the location you want to edit on OpenStreetMap.org and click *Edit* at the top
```

biodranik commented on content/faq/editing/advanced-map-editing/index.md:
```diff
@@ -0,0 +1,79 @@
+For editing OpenStreetMap with Go Map follow these steps:
+
+1. Download the app from the [App Store](https://apps.apple.com/de/app/go-map/id592990211)
+2. Follow the detailed [LearnOSM tutorial](https://learnosm.org/en/mobile-mapping/gomap/)
```
Can these tutorial hints be embedded above, where these editors are referenced? Or maybe a separate section "Tutorials" would be better?

map-per replied:
:+1: I moved the tutorial hints to the editor overview

biodranik commented on content/faq/editing/advanced-map-editing/index.md:
```diff
@@ -0,0 +1,79 @@
+1. Download the app from the [Play Store](https://play.google.com/store/apps/details?id=de.blau.android) or from [F-Droid](https://f-droid.org/en/packages/de.blau.android/).
+2. Follow the detailed [LearnOSM tutorial](https://learnosm.org/en/mobile-mapping/vespucci/)
+
+## What happens with my edits?
```
Need to clarify when edits appear in Organic Maps

biodranik commented on content/faq/editing/advanced-map-editing/index.md:
```diff
@@ -0,0 +1,79 @@
+
+## Community and Wiki
+
+OpenStreetMap is a community. If you need help you can ask in the [OSM Forum](https://community.openstreetmap.org/) or take a look at the [OSM Wiki](https://wiki.openstreetmap.org/) documentation.
```
Any live discussion channels? List of telegram/other messenger groups? Remember, our audience is not a geeky one.

map-per replied:
The help and support page in the OSM Forum is the main place for questions. But there are also other channels (see: https://wiki.openstreetmap.org/wiki/Contact_channels). I haven't tried out any of them though.

biodranik commented on content/faq/editing/advanced-map-editing/index.md:
```diff
@@ -0,0 +1,79 @@
+    > You can also open anonymous Notes, but this is not recomended as you won't get notified when the issue is solved or there are further questions.
+
+
+2. Zoom to the map location on [OpenStreetMap.org](https://www.openstreetmap.org) and press *Add a note to the map* (second icon from the bottom on the right menue). Then drag the blue map marker to the exact location.
```
Is this numbered list properly rendered by Zola?

map-per replied:
Added &nbsp; to have more space between the quote and the next list element

biodranik replied:
Please don't do that. No visual formatting with the content. It should be done with CSS, I've pushed a fix.

biodranik commented on content/faq/editing/advanced-map-editing/index.md:
```diff
@@ -0,0 +1,79 @@
+In the ID Editor, you can see the Tags that ID is hiding from you by expanding the *Tags* section in the *Edit feature* side panel.
+
+## OSM Notes
+If you don't have time or the problem is too complicated for editing the OSM data yourself OSM Notes ([Wiki](https://wiki.openstreetmap.org/wiki/Notes)) are the way to go. You can place such a note in the location of the map error and describe the problem in detail. Other OSM volunteers can then help and solve the issue. You will get e-mail notifications via your OSM account in case they have fur ...
```
It's better to leave an empty line after each header.

biodranik reviewed (commented):
Thanks! A few comments.

map-per commented:
Updated the faq page and included your suggestions.

biodranik commented:
@map-per FYI for the future PRs: with a write access to the repo, if you push a branch here, then its preview can be available for anyone.

biodranik commented:
We can proceed with the PR to unblock other tasks, although the text already looks too complicated for a non-geeky, non-OSM user. It would be great to stick to the most important, precise, succinct info for new users on the initial page, and move any details for those who are interested to a different page. Otherwise, there is a risk that your grandma just closes the long web page with a lot of text without reading it, and won't proceed with any contributions.

You can test it with OM + a link to this page combo with any (non-geeky) person who didn't know about OM and OSM before.

map-per commented:
I think it's OK that this page explains more difficult content, as it is about **advanced** editing.

For **simple** editing with OM we should create a separate faq page.

biodranik commented:
There is an important UX flow nuance: _all_ OM users (including complete newbies) will see the related message when they try to add something on the map and won't find the interested object.

That means that the landing page should be easy to digest for _all_ OM users. Those who is interested in more advanced details, may click to read them.

map-per commented:
> We can proceed with the PR to unblock other tasks

Then let's merge this and do further improvements in the next iterations

biodranik reviewed (approved):
Ok, any ideas on a web page for grannies?

map-per commented:
We could move community, wiki and data model to a dedicated OpenStreetMap page. And as soon as we have an Organic Maps editing page (for grannies) we could move the OSM Notes section there.

I think, for the FAQ, the Organic Maps editing page is the highest priority now.
